modules/waypoint_names_to_coordinates.py

```diff
@@ -0,0 +1,59 @@
+def waypoint_names_to_coordinates(waypoint_names: list[str],
```

Comment:
Put the type annotations in double quotation marks as Python 3.8 doesn't support type annotating containers without importing the typing modules (and we want to avoid importing the typing module).